tests/specs/erc20/concrete-rules.txt

```diff
@@ -1,27 +1,28 @@
 EDSL.#ceil32
 EDSL.keccakIntList
-EVM-TYPES.#signed.positive
-EVM-TYPES.#signed.negative
-EVM-TYPES.#unsigned.positive
-EVM-TYPES.#unsigned.negative
-EVM-TYPES.powmod.nonzero
-EVM-TYPES.powmod.zero
-EVM-TYPES.signextend.invalid
-EVM-TYPES.signextend.negative
-EVM-TYPES.signextend.positive
-EVM-TYPES.#asWord.recursive
-EVM-TYPES.#asByteStack
-EVM-TYPES.#asByteStackAux.recursive
-EVM-TYPES.#padToWidth
-EVM-TYPES.#padRightToWidth
-EVM.#memoryUsageUpdate.some
+EVM.Cextra
 EVM.Cgascap
+EVM.Cmem
 EVM.Csstore.new
 EVM.Csstore.old
+EVM.#memoryUsageUpdate.some
 EVM.Rsstore.new
 EVM.Rsstore.old
-EVM.Cextra
-EVM.Cmem
+EVM-TYPES.#asByteStack
+EVM-TYPES.#asByteStackAux.recursive
+EVM-TYPES.#asWord.recursive
+EVM-TYPES.#padRightToWidth
+EVM-TYPES.#padToWidth
+EVM-TYPES.powmod.nonzero
+EVM-TYPES.powmod.zero
+EVM-TYPES.#range
+EVM-TYPES.#signed.negative
+EVM-TYPES.#signed.positive
+EVM-TYPES.signextend.invalid
+EVM-TYPES.signextend.negative
+EVM-TYPES.signextend.positive
+EVM-TYPES.#unsigned.negative
+EVM-TYPES.#unsigned.positive
```

Comment:
Is this just a reordering change?

Reply:
OK. You reordered and also added `EVM-TYPES.#range`, which is good.

Indeed, note that we have the following three additional items in the concrete-rule.txt file, in the latest proofs:
```
EVM-TYPES.mapWriteBytes.recursive
EVM-TYPES.#lookup.some
EVM-TYPES.#lookup.none
```

I already suggested to add `EVM-TYPES.mapWriteBytes.recursive` in another comment.

The remaining two items for lookup may not be needed. So you can just ignore them for now.

Reply:
I'll add the recursive `mapWriteBytes` lemma, and see if the other two are needed. My guess is not for now, but maybe when you start adding more regression tests for downstream proofs.

Reply:
And yes, it was just a reordering.